python/tank/context.py

```diff
@@ -89,26 +89,32 @@ def __repr__(self):
         return "<Sgtk Context: %s>" % ("\n".join(msg))
 
     def __str__(self):
-        # smart looking string representation
-        
+        """
+        String representation for context
+        """
         if self.project is None:
             # We're in a "site" context, so we'll give the site's url
             # minus the "https://" if that's attached.
             ctx_name = self.shotgun_url.split("//")[-1]
         
         elif self.entity is None:
-            # project-only!
-            ctx_name = "%s" % self.project.get("name")
+            # project-only, e.g 'Project foobar'
+            ctx_name = "Project %s" % self.project.get("name")
```

Comment:
this is the main code change, everything else is just PEP tweaks/cleanup and tests.